roles/openshift-defaults/defaults/main.yml

```diff
@@ -6,4 +6,12 @@ openshift_password: ""
 openshift_project_name: ""
 openshift:
   common:
-    client_binary: "oc"
+    client_binary: "oc"  
+docker:
+  common:
+    client_binary: "docker"
+  registry:
+    host: registry.env3-1.innovation.labs.redhat.com
```

Comment:
I think we sooner than later need to find a solution avoid having hostnames/FQDNs in the source tree - this makes it quite hard to work on new environments.